src/app/components/GearCardHeader/GearCardHeader.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import DrumMachineIcon from '../assets/DrumMachineIcon';
+import SynthesizerIcon from '../assets/SynthesizerIcon';
+import EffectIcon from '../assets/EffectIcon';
+import Typography from '../Typography/Typography';
+import style from './GearCardHeader.module.css';
+
+type GearCardHeaderProps = {
+  type: 'Drum' | 'Effect' | 'Synth';
+  name: string;
+};
+
+export default function GearCardHeader({
+  type,
+  name,
+}: GearCardHeaderProps): JSX.Element {
+  if (type === 'Synth') {
+    return (
+      <header className={style.header}>
+        <div>
+          <SynthesizerIcon fill="var(--navigation-icon-color)" />
+        </div>
+        <div>
+          <Typography size="m" color="green" children={name} />
+        </div>
+      </header>
+    );
+  } else if (type === 'Effect') {
+    return (
+      <header className={style.header}>
+        <div>
+          <EffectIcon fill="var(--navigation-icon-color)" />
+        </div>
+        <div>
+          <Typography size="m" color="green" children={name} />
+        </div>
+      </header>
+    );
+  } else {
+    return (
+      <header className={style.header}>
+        <div>
+          <DrumMachineIcon fill="var(--navigation-icon-color)" />
+        </div>
+        <div>
+          <Typography size="m" color="green" children={name} />
+        </div>
+      </header>
+    );
+  }
```

Comment:
Please consider a map for this conditional rendering. Like this: https://github.com/marwinburesch/moviescout/blob/6dbae1cc80f17ea1ac28af0d9288ccf4b65026f6/src/app/components/MovieButton/MovieButton.tsx#L27

Reply:
Resolved in [e95b67b](https://github.com/Seborider/plugs./pull/31/commits/e95b67ba2f02efebbd35d48511f854335417eb81)